database/migrations/Version20260424120000.php

```
@@ -0,0 +1,44 @@
+
+/**
+ * Class Version20260424120000
+ * @package Database\Migrations
+ *
+ * Enforce uniqueness of (user_id, device_identifier) on user_trusted_devices.
+ * Replaces the plain index utd_user_device_idx with a unique one so that a
+ * given user can never accumulate duplicate device rows.
+ */
+final class Version20260424120000 extends AbstractMigration
+{
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)